$NetBSD: patch-ad,v 1.7 2017/09/28 10:40:35 spz Exp $

- netbsd has stdint.h
- remove stray commas
- make sure hardening flags can be inherited

--- build/Makefile.NetBSD.orig	2012-08-01 08:48:15.000000000 +0000
+++ build/Makefile.NetBSD
@@ -1,7 +1,7 @@
 # Xymon compile-time settings for NetBSD systems
 # From Emmanuel Dreyfus.
 #
-OSDEF = -DBSD
+OSDEF = -DBSD -DHAVE_STDINT_H
 
 # NETLIBS: None needed
 NETLIBS =
@@ -11,17 +11,17 @@ PKGDIR?=/usr/pkg
 CC= gcc
 GCCVER := $(shell gcc -dumpversion|cut -d. -f1)
 ifeq ($(GCCVER),4)
-   CFLAGS = -g -O2 -Wall -Wno-unused -Wno-pointer-sign -D_REENTRANT $(LFSDEF) $(OSDEF) \
-       -I${PKGDIR}/include -L${PKGDIR}/lib, -Wl,--rpath=${PKGDIR}/lib
+   CFLAGS += -g -O2 -Wall -Wno-unused -Wno-pointer-sign -D_REENTRANT $(LFSDEF) $(OSDEF) \
+       -I${PKGDIR}/include -L${PKGDIR}/lib -Wl,--rpath=${PKGDIR}/lib
 else
-   CFLAGS = -g -O2 -Wall -Wno-unused -D_REENTRANT $(LFSDEF) $(OSDEF) \
-       -I${PKGDIR}/include -L${PKGDIR}/lib, -Wl,--rpath=${PKGDIR}/lib
+   CFLAGS += -g -O2 -Wall -Wno-unused -D_REENTRANT $(LFSDEF) $(OSDEF) \
+       -I${PKGDIR}/include -L${PKGDIR}/lib -Wl,--rpath=${PKGDIR}/lib
 endif
 RPATH = "-Wl,--rpath,"
 
 # Compile flags for debugging
 # CFLAGS = -g -DDEBUG -Wall -D_REENTRANT $(LFSDEF) $(OSDEF) \
-       -I${PKGDIR}/include -L${PKGDIR}/lib, -Wl,--rpath=${PKGDIR}/lib
+       -I${PKGDIR}/include -L${PKGDIR}/lib -Wl,--rpath=${PKGDIR}/lib
 
 # Mail program: This must support "CMD -s SUBJECT ADDRESS" to send out a mail with a subject
 # Typically, this will be "mail" or "mailx"
